html-forms/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="index.html">
+    <label for="email">Email </label>
+    <div><input type="text" id="email"></div>
+    <label for="coding">Why is coding cool?</label>
+    <div><select name="coding">
+  <option value="selectoption">Select an option</option>
+  <option value="interesting">Because it's interesting</option>
+  <option value="fun">Because it's fun</option>
+  <option value="powerful">Because it's powerful</option>
+      </select>
+      <div><label for="favoritelanguage">Favorite Language (choose one)</label></div>
+      <br />
+      <input type="radio" id="html" name="favoritelanguage" /> HTML
+      <input type="radio" id="css" name="favoritelanguage" /> CSS
+      <input type="radio" id="javascript" name="favoritelanguage" /> JavaScript
+      <div><label for="signup">Would you like to sign up for more information?</label></div>
+      <br />
+      <input type="checkbox" name="signmeup" id="signup" /> Sign me up!
+      <div><label for="submit">Comments</label></div>
```

Comment:
This `label` element's `for` attribute does not match the `id` attribute on the `textarea` it is paired with